homeassistant/components/samsungtv/bridge.py

```diff
@@ -655,7 +658,7 @@ def _remote_event(self, event: str, response: Any) -> None:
 
     async def _async_send_power_off(self) -> None:
         """Send power off command to remote."""
-        if self._get_device_spec("FrameTVSupport") == "true":
+        if str(self._get_device_spec("FrameTVSupport")) == "true":
```

Comment:
This change is unnecessary.
```suggestion
        if self._get_device_spec("FrameTVSupport") == "true":
```

Reply:
Even with no changes, this line fails the required mypy linting. This change felt like the simplest way to address it, but open to alternative suggestions!

<img width="1383" alt="image" src="https://github.com/home-assistant/core/assets/31580846/e191e8db-49aa-4f40-a893-8d623d2fed3f">

Reply:
This was due to the "async" change you made earlier. I don't think it's necessary anymore, so please remove.